views/default/river/elements/body.php

```diff
@@ -53,21 +54,9 @@
 	$responses = "<div class=\"elgg-river-responses\">$responses</div>";
 }
 
-$group_string = '';
-$object = $item->getObjectEntity();
-$container = $object->getContainerEntity();
-if ($container instanceof ElggGroup && $container->guid != elgg_get_page_owner_guid()) {
-	$group_link = elgg_view('output/url', [
-		'href' => $container->getURL(),
-		'text' => $container->name,
-		'is_trusted' => true,
-	]);
-	$group_string = elgg_echo('river:ingroup', [$group_link]);
-}
-
 echo <<<RIVER
 $menu
-<div class="elgg-river-summary">$summary $group_string <span class="elgg-river-timestamp">$timestamp</span></div>
+<div class="elgg-river-summary">$summary <span class="elgg-river-timestamp">$timestamp</span></div>
```

Comment:
Why? This would push the last summary word into the timestamp. "Foo create the page Bar15 seconds ago." Bad for accessibility.